### ISSUE TITLE
Error 500

### ISSUE THREAD

**Issue by benjisympa** [owncloud/mail#1201]
Hello, thanks for your jobs but I follow your instructions : 

- Download (wget "https://en0ch.se/index.php/s/3v5J3XXeQCCp73p/download?path=%2F&files=owncloud_mail_nightly_build_20160102.tar.gz")

- Extract the tar archive to 'path-to-owncloud/apps' (tar -xvf in /usr/share/owncloud)

- Navigate to »Apps«, choose the category »Productivity«, find the Mail app and enable it.

But when I go to the application mail on owncloud I have an error 500 :/, I don't know why.

Thanks for your response.

**Comment by ChristophWurst** [comment]
Thanks for reporting. Please fill out the [issue template](https://raw.githubusercontent.com/owncloud/core/master/issue_template.md). The owncloud log (data/owncloud.log) is especially important, because the error is likely to be logged there. Thanks.

**Comment by benjisympa** [comment]
Thank you for your answer.
Ok, I will do this.
Can I send you private email ?

**Comment by ChristophWurst** [comment]
> Can I send you private email ?

Please remove any sensitive information if there is any and post the information here. It not necessary to post the full log. Simple try to reproduce the bug and post the last few lines here. This way any developer can have a look at this bug.

**Comment by benjisympa** [comment]
### Steps to reproduce
Download (wget "https://en0ch.se/index.php/s/3v5J3XXeQCCp73p/download?path=%2F&files=owncloud_mail_nightly_build_20160102.tar.gz")

Extract the tar archive to 'path-to-owncloud/apps' (tar -xvf in /usr/share/owncloud)

Navigate to »Apps«, choose the category »Productivity«, find the Mail app and enable it.

### Expected behaviour
Launch the mail application

### Actual behaviour
Error 500 with white page. I can't configure smtp, I just clicked on the enable button.

### Server configuration
**Operating system**: bananian (débian for bananapi in unstable/sid version)

**Web server:** apache2

**Database:** mariadb

**PHP version:** 

**ownCloud version:** (see ownCloud admin page)
ownCloud 7.0.12~dfsg-1 (Debian) (stable)
Developed by the ownCloud community, the source code is licensed under the AGPL.

**Updated from an older ownCloud or fresh install:**
fresh install

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```
Enabled:
 - activity
 - bookmarks
 - calendar
 - contacts
 - documents
 - files
 - files_pdfviewer
 - files_sharing
 - files_texteditor
 - files_trashbin
 - files_versions
 - files_videoviewer
 - firstrunwizard
 - gallery
 - mail
 - search_lucene
 - templateeditor
Disabled:
 - external
 - files_encryption
 - files_external
 - user_external
 - user_ldap
 - user_webdavauth

**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** yes/no
Yes

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No

#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your ownCloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:**
Chromium

Insert your ownCloud log here

Error	PHP	Call to undefined method OC\Server::getSystemConfig() at /usr/share/owncloud/apps/mail/templates/index.php#6	2016-01-02T20:34:30+00:00
Error	PHP	Call to undefined method OC\Server::getSystemConfig() at /usr/share/owncloud/apps/mail/templates/index.php#6	2016-01-02T20:31:00+00:00
Error	PHP	Call to undefined method OC\Server::getSystemConfig() at /usr/share/owncloud/apps/mail/templates/index.php#6	2016-01-02T20:28:10+00:00
Error	PHP	Call to undefined method OC\Server::getSystemConfig() at /usr/share/owncloud/apps/mail/templates/index.php#6	2016-01-02T20:27:03+00:00
Error	PHP	Call to undefined method OC\Server::getSystemConfig() at /usr/share/owncloud/apps/mail/templates/index.php#6	2016-01-02T20:26:39+00:00
Error	PHP	Call to undefined method OC\Server::getSystemConfig() at /usr/share/owncloud/apps/mail/templates/index.php#6	2016-01-02T20:26:36+00:00
Error	PHP	Automatically populating $HTTP_RAW_POST_DATA is deprecated and will be removed in a future version. To avoid this warning set 'always_populate_raw_post_data' to '-1' in php.ini and use the php://input stream instead. at Unknown#0

**Comment by ChristophWurst** [comment]
We're using a private API of core (``OC\Server::getSystemConfig``). Apparently it is not available on owncloud 7.

**Comment by benjisympa** [comment]
Ok, I will try with owncloud 8 directly from git not from debian package, thanks :)

**Comment by ChristophWurst** [comment]
> Ok, I will try with owncloud 8 directly from git not from debian package, thanks :)

I recommend using an official release if it's a production system. There are packages for debian: https://owncloud.org/install/#instructions-server

**Comment by ChristophWurst** [comment]
@benjisympa I fixed the error in #1248. Are you still using owncloud 7?

**Comment by Gomez** [comment]
To get cleanly fixed it needs this branch and this one: mimetypeicon-version-hack. Both merged now.
:+1:
